js/dimensions.js

```diff
@@ -1,4 +1,7 @@
 $(document).ready(function() {
+  const SMALL_CROPPING_WIDTH_LIMIT = 800;
```

Comment:
Can you add a comment specifying why we have a limit? or maybe a link to the issue.